charts/camunda-platform/templates/NOTES.txt

```diff
@@ -142,3 +142,58 @@ Default user and password: "{{ .Values.identity.firstUser.username -}}/{{ .Value
 {{ end }}
 
 {{- end }}
+
+{{- if or (.Values.global.ingress.enabled) (.Values.identity.ingress.enabled) (.Values.operate.ingress.enabled) (.Values.optimize.ingress.enabled) (.Values.tasklist.ingress.enabled) ( index .Values "zeebe-gateway" "ingress" "enabled" ) (.Values.webModeler.ingress.enabled) }}
+
+### Ingress
+
+{{- if .Values.global.ingress.enabled }}
+
+#### Combined Ingress
+
+- Ingress:
+  - Enabled: {{ .Values.global.ingress.enabled }}
+  - Web Applications:
+    - Identity: https://{{ .Values.global.ingress.host }}{{ .Values.identity.contextPath }}
+    - Operate: https://{{ .Values.global.ingress.host }}{{ .Values.operate.contextPath }}
+    - Optimize: https://{{ .Values.global.ingress.host }}{{ .Values.optimize.contextPath }}
+    - Tasklist: https://{{ .Values.global.ingress.host }}{{ .Values.tasklist.contextPath }}
+    - WebModeler: https://{{ .Values.global.ingress.host }}{{ .Values.webModeler.contextPath }}
+  - Keycloak Authentication: https://{{ .Values.global.ingress.host }}/auth
+    {{- if index .Values "zeebe-gateway" "ingress" "enabled" }}
+  - Zeebe Gateway: grpc://{{ index .Values "zeebe-gateway" "ingress" "host" }}
+    {{- end }}
+
+{{- end }}
+
+{{- if or (.Values.identity.ingress.enabled) (.Values.operate.ingress.enabled) (.Values.optimize.ingress.enabled) (.Values.tasklist.ingress.enabled) ( index .Values "zeebe-gateway" "ingress" "enabled" ) (.Values.webModeler.ingress.enabled) }}
+
+#### Separated Ingress
+
+- Ingress:
+  - Web Applications:
+    {{- if index .Values.identity.ingress.enabled }}
```

Comment:
No need to use the `index` Helm function. It's used only with `zeebe-gateway` because it has a "dash" in it.